src/conversations/ConversationsCard.js

```diff
@@ -82,7 +80,9 @@ class ConversationsCard extends PureComponent<Props> {
           />
         </View>
         {conversations.length === 0 ? (
-          <Label style={componentStyles.emptySlate} text="No recent conversations" />
+          <View style={styles.emptyStateContainer}>
```

Comment:
Is this extra wrapper required?
Can't we do it with `Label` itself?

Reply:
Label does not return text in a view, it just returns a `Text` element so the wrapper is required.

Reply:
Why do you need a wrapper though?

Reply:
To align using `justifyContent`.
Also `SearchEmtpyState` uses a wrapper View, and I moved the styles of that view to miscStyles and reused it here.